src/styles/globals.css

```
@@ -2,6 +2,13 @@
 @tailwind components;
 @tailwind utilities;
 
+@font-face {
+  font-family: 'Pretendard-Regular';
+  src: url('https://fastly.jsdelivr.net/gh/Project-Noonnu/noonfonts_2107@1.1/Pretendard-Regular.woff') format('woff');
+  font-weight: 400;
+  font-style: normal;
+}
+
 :root {
   font-size: 62.5%;
   color: white;
```

p4: 현재 프로젝트에서 폰트를 적용한다면, `globals.css` 파일에서 폰트를 모든 페이지에 적용 시키는 방법은 어떠신가요~~?
( 물론 예상치 못하게 상속받지 않는 요소가 있을 수 있기에, `tailwind.config.ts`에 폰트를 지정해놓는것은 좋은것 같습니다!!)

```suggestion
@tailwind components;
@tailwind utilities;

@font-face {
  font-family: 'Pretendard-Regular';
  src: url('https://fastly.jsdelivr.net/gh/Project-Noonnu/noonfonts_2107@1.1/Pretendard-Regular.woff') format('woff');
  font-weight: 400;
  font-style: normal;
}

:root {
  font-size: 62.5%;
  font-family: 'Pretendard-Regular';
  color: white;
}
```

---

구웃 반영하겠슴다~